### ISSUE TITLE
[low] JAWS+Chrome: Many Pages: The main region is not marked up semantically

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5450]
## Action Performed:
1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Press Q key to navigate to the main landmark and verify the announcement.

## Expected Result:
The main landmark of the page needs to be marked up semantically, the focus should move to the main landmark once pressing the Q key.

## Actual Result:
"There is no main region on this page" message is announced by the screen reader once pressing the Q key.

Other occurrences:
1. Terms of service -> No landmarks are defined on the page.
2. Licenses -> No landmarks are defined on the page.
3. Set password -> No landmarks are defined on the page.
4. Email verification -> No landmarks are defined on the page.

## Workaround:
No workaround

## Area issue was found in:
Landing page, Terms of service page, Licenses page, Set password page, Email verification page

## Failed WCAG checkpoints
2.4.1

## User impact:
The screen reader users are not given the possibility to navigate directly to the main page content using landmarks navigation.

## Suggested resolution:
Wrap the main page content into the <main> tag, a similar functionality is available using the ARIA attribute role="main". It is best practice to programmatically identify various sections of the page (e.g. header, navigation, main, footer, etc.) in order to provide an easy access to these sections for the screen reader users. See: https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134521456-948193d7-1a11-4d0d-a79d-e7d8203c02c0.mp4

https://user-images.githubusercontent.com/88733897/134521475-eb893d6e-5225-47cc-a8fa-322cc6c9023b.mp4

https://user-images.githubusercontent.com/88733897/134521489-8c90be45-3680-4f09-8229-8a3a8e84fd5f.mp4

https://user-images.githubusercontent.com/88733897/134521499-c4c2ccce-b2fc-4367-b0ae-7e223283119a.mp4

https://user-images.githubusercontent.com/88733897/134521512-0f6124a9-42cf-4acb-997e-82825fa4f169.mp4

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
